### PR TITLE
Ruby Project: Event Manager: Rewrite Initial Setup opening paragraph

### PR DESCRIPTION
**Pull request by Maefire**
## Because

In the previous TOP Curriculum, throughout Foundations, it is assumed that all projects are cloned into your repos directory. Since the Ruby and JS paths are meant to take place after Foundations, it is assumed that you will follow previous practices. This means cloning the repo and knowing the basic terminology and commands. I don't recall if mkdir or touch was covered, but the cloning should be second nature at this point, and I think knowing what CLI stands for would also be a part of that knowledge.

## This PR

- reword the Initial Setup section of the Project: Event Manager to better fit previous curriculum flow

## Additional Information

[Discord Message Link](https://discord.com/channels/505093832157691914/505093832157691916/1080054550356045894)

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: ruby/files_and_serialization/project_event_manager.md [hunk 1]
```diff
@@ -28,14 +28,10 @@ representatives for each attendee based on their zip code.
 
 ### Initial Setup
 
-Create a project directory folder named `event_manager` wherever you want to store
-your project. In the project directory, create another folder named `lib` and inside
-that folder create a plain text file named `event_manager.rb`. Using your command-line
-interface (CLI), you can enter the following commands:
+Create a new GitHub repository named `event_manager` and clone it to your `repos` directory. In the new `event_manager` directory, create another folder named `lib` and inside
```
Comment (KevinMulhern) [inline ruby/files_and_serialization/project_event_manager.md:31]: Whats your thoughts on having this as a numbered list of steps?

```suggestion
1. Create a new GitHub repository named `event_manager` and clone it to your `repos` directory. 
2. In the new `event_manager` directory, create another folder named `lib` and inside that folder create a plain text file named `event_manager.rb`. You can create the directory and file by using the following commands:
```

File: ruby/files_and_serialization/project_event_manager.md [hunk 2]
```diff
@@ -28,14 +28,10 @@ representatives for each attendee based on their zip code.
...
+that folder create a plain text file named `event_manager.rb`. You can create the directory and file by using the follow commands:
```
Comment (KevinMulhern) [inline ruby/files_and_serialization/project_event_manager.md:32]: Just a minor typo

```suggestion
that folder create a plain text file named `event_manager.rb`. You can create the directory and file by using the following commands:
```

**Review by KevinMulhern — CHANGES_REQUESTED** [review]

File: ruby/files_and_serialization/project_event_manager.md (hunk 1 above)
Comment (Maefire) [reply ruby/files_and_serialization/project_event_manager.md:31]: That would work, in my opinion. While a little different than the normal, it still addresses the main concern I had, and follows most of the conventional TOP lesson patterns.

File: ruby/files_and_serialization/project_event_manager.md [hunk 3]
```diff
@@ -28,14 +28,11 @@ representatives for each attendee based on their zip code.
 
 ### Initial Setup
 
-Create a project directory folder named `event_manager` wherever you want to store
-your project. In the project directory, create another folder named `lib` and inside
-that folder create a plain text file named `event_manager.rb`. Using your command-line
-interface (CLI), you can enter the following commands:
+1. Create a new GitHub repository named `event_manager` and clone it to your `repos` directory. 
+2. In the new `event_manager` directory, create another folder named `lib` and inside that folder create a plain text file named `event_manager.rb`. You can create the directory and file by using the following commands:
+that folder create a plain text file named `event_manager.rb`. You can create the directory and file by using the follow commands:
```
Comment (KevinMulhern) [inline ruby/files_and_serialization/project_event_manager.md:33]: Just some leftovers that need removed.
```suggestion
```

**Review by KevinMulhern — CHANGES_REQUESTED** [review]

**Review by KevinMulhern — APPROVED** [review]
Thanks for doing this @Maefire 💪

**Pull request merged**